src/index.js

```diff
@@ -3,22 +3,23 @@ import bodyParser from 'body-parser';
 import cors from 'cors';
 import dotenv from 'dotenv';
 import swaggerUI from 'swagger-ui-express';
-import testRoute from './routes/index';
 import docs from '../swagger.json';
+import indexRouter from './routers';
+import { startServerConnection } from './server';
 
 dotenv.config();
 
-const PORT = process.env.PORT || 4000;
-
 // Create global app object
 const app = express();
+app.set('port', parseInt(process.env.PORT || 4000, 10));
+app.set('json spaces', 2);
 
 // Express config defaults
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 app.use(cors());
 
-app.get('/', testRoute);
+app.use('/', indexRouter);
 
 // route for API documentation
 app.use('/docs', swaggerUI.serve, swaggerUI.setup(docs));
```

Comment:
Server initialization should be in the server file. You should move everything about app inside the server file and wrap it in a non-blocking function. I mean you should use promise. Let's sync on this so we don't waste too much time @micah-akpan

Reply:
Okay, I will start this now, and reach out to you if I have any blocker